docs/discord/development.md

```diff
@@ -256,7 +256,7 @@ thank you clarifying that 😁
 Yes, that sounds good.
 
 ##### black\_hammer\_67 06/24/2020 14:04:12
-ok i see, so I am looking for points that they have a certain value on y axis > 0.1 for example ?
+ok i see, so I am looking for points that they have a certain value on y-axis > 0.1 for example ?
```

Comment:
This file shout not be changed manually, the next synchronization will cause conflicts because of this manual change.